crates/nu-command/src/platform/ansi/strip.rs

```diff
@@ -0,0 +1,123 @@
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::Call, ast::CellPath, engine::Command, engine::EngineState, engine::Stack, Category,
+    Example, PipelineData, ShellError, Signature, Span, SyntaxShape, Value,
+};
+use strip_ansi_escapes::strip;
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "ansi strip"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("ansi strip")
+            .rest(
+                "column path",
+                SyntaxShape::CellPath,
+                "optionally, remove ansi sequences by column paths",
+            )
+            .category(Category::Platform)
+    }
+
+    fn usage(&self) -> &str {
+        "strip ansi escape sequences from string"
+    }
+
+    fn run(
+        &self,
+        engine_state: &EngineState,
+        stack: &mut Stack,
+        call: &Call,
+        input: PipelineData,
+    ) -> Result<nu_protocol::PipelineData, ShellError> {
+        operate(engine_state, stack, call, input)
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "strip ansi escape sequences from string",
+            example: "echo [(ansi gb) 'hello' (ansi reset)] | str collect | ansi strip",
+            result: None,
```

Comment:
Code looks good - you could even add a result in here that's the result of the `ansi strip`'d string

Reply:
No problem - sorry!